master/internal/project/postgres_project.go

```diff
@@ -29,7 +29,7 @@ const (
 	// MaxRetries is the maximum number of retries for transaction conflicts.
 	MaxRetries = 5
 	// ProjectKeyRegex is the regex pattern for a project key.
-	ProjectKeyRegex = "^[A-Z0-9]{1,5}$"
+	ProjectKeyRegex = "^[a-zA-Z0-9]{1,5}$"
```

Comment:
Do we want to allow lower case keys? Does this now mean that keys are case sensitive? My original thought was that we just capitalize the characters

Reply:
Keys are case insensitive. I changed the validation to allow for folks to submit keys in whatever case they'd like, it gets stored & displayed in uppercase.

But I see you're point if someone is using mixed cases to submit a key that could be confusing.

Reply:
This is a good point in terms of understanding where the responsibility for preparing this information should lie. Is it the responsibility of the request to make sure that the keys are in uppercase (i.e. before sending the request, capitalizing the values or should be be done in `postgres_project.go` prior to querying the db).

I had been of the mindset to allow the requested key to come in as any case; however, that could be misunderstood as allowing case sensitive keys. 

Based on that, I think it makes more sense to require that the request to be responsible for ensuring that the incoming key is in the correct format & returning an error if it's not so it's abundantly clear that keys are required to be capitalized.

Reply:
Ahh I see so this is just for validation. I think that is fine then

Reply:
Maybe a warning message perhaps. But it may just be easier to force capital letters

Reply:
I think even just having some sort of note that users can see in the project creation/edit modal next to the text field for the project key letting them know that it's case insensitive will be the best of both worlds